docs/server-side/v2/how-to-guides/live-transcription-hls.mdx

```diff
@@ -0,0 +1,213 @@
+
+4. Select 'Yes' to the question 'Do you require live transcription for your live streams?'.
+![select yes to live transcription](/docs/v2/create-template-live-transcription-question.png)
+
+5. Deploy the template and proceed further. On the final screen, use the different roles available to experience the 
+
+Read up more on the live streaming experience [here](/get-started/v2/get-started/features/live-streaming).
+
+
+
+#### Editing an existing template
+1. Go to the template configuration
+    
+2. Click on ‘Transcription (Beta)’ tab.
+
+3. Enable ‘Live Transcription’ toggle.
+
+![enable live transcription](/docs/v2/live-transcription-enable-toggle.png)
+
+</Tab>
+
+
+<Tab id='implementation-1'>
+You can refer to our [Live Streaming API documentation](/server-side/v2/api-reference/live-streams/start-live-stream-for-room) to start recording.
+
+1. You are required to mark the transcription `enabled` flag in the request as `true` and select `live` in `modes`to transcribe.
+2. You can override the live transcription settings given in the template when starting the live stream through the Live Stream API.
+3. If no transcription input is given within the API request, the API will pick up the transcription configuration from the template policy / configuration by default when starting the live stream.
+
+
+</Tab>
+
+Live stream can be started using the Live Streaming REST API, 100ms SDK or through 100ms Prebuilt.
+
+
+#### Advanced Configuration
+
+There are two advanced configuration that are currently offered.
+
+1. **Custom Vocabulary**: Add non-dictionary words like names, abbreviations, slang, and technical jargon which may not be recognised by the AI model for better transcription accuracy.
+2. **Language:** Configure the primary spoken language that has to be transcribed. This will hint the AI model to perform transcription more accurately.  Currently, only **English** is supported, which is the default. Support for more languages will follow soon.
+
+![advanced transcription config](/docs/v2/advanced-transcription-config.png)
+
+
+### Consumption
+
+#### Prebuilt
+Our Prebuilt player on web supports HLS with live transcription out of the box. It allows the functionality of enabling and disabling this feature using the `CC` button on the player.
+
+<video loop="true" autoplay="autoPlay" controls="false" id="vid" muted>
+    <source src="/docs/docs/v2/cc-on-hls-player-web.mp4" type="video/mp4" />
+</video>
+
+Learn more about live streaming [here](/get-started/v2/get-started/features/live-streaming) and get started with integrating Prebuilt using [this](/get-started/v2/get-started/prebuilt/overview).
+
+#### Within your player
+
+The transcript `.vtt` files are delivered as part of the HLS manifest itself. If a HLS compatible player is used, it should support closed captioning by default.
+
+All that is required is to input the URL to `master.m3u8` into a compatible HLS player.
```

Comment:
instead of specifying master.m3u8 - phrase it as the master manifest. the exact file name may be changed later, its not a fixed.